docs/resources/mcs_db_cluster.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: "mcs"
+page_title: "mcs: db_cluster"
+sidebar_current: "docs-db-cluster"
+subcategory: ""
+description: |-
+  Manages a db cluster.
+---
+
+# mcs\_db\_cluster (Resource)
+
+Provides a db cluster resource. This can be used to create, modify and delete db cluster for galera_mysql and postgresql datastores.
+
+## Example Usage
+
+```terraform
+
+resource "mcs_db_cluster" "mydb-cluster" {
```

Comment:
sort keys in this block to have the same order with reference

Reply:
Fixed